format/spec.md

```diff
@@ -1312,7 +1325,7 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**       | **`JSON object by field ID`**             | `{"1": 1, "2": "bar"}`                     | Stores struct fields using the field ID as the JSON field name; field values are stored using this JSON single-value format |
 | **`list`**         | **`JSON array of values`**                | `[1, 2, 3]`                                | Stores a JSON array of values that are serialized using this JSON single-value format |
 | **`map`**          | **`JSON object of key and value arrays`** | `{ "keys": ["a", "b"], "values": [1, 2] }` | Stores arrays of keys and values; individual keys and values are serialized using this JSON single-value format |
-
+| **`geometry`**     | **`JSON string`**                         |`00000000013FF00000000000003FF0000000000000`| Stores WKB as a hexideciamal string, see [Appendix G](#appendix-g-geospatial-notes)                                                                                                                                                                                                                      |
```

Comment:
@rdblue am not entirely sure where this part of the spec is used.  Should it also match the above (the more optimized serialization for stats)?

Reply:
For this, I'd probably use a list or object of numbers. Would that work?

Reply:
This is used for default values and for encoding values in JSON expressions for filtering.

What about using WKT here instead of WKB?

Reply:
Good idea, added.